cmd/todo_mr.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var todoMRCmd = &cobra.Command{
+	Use:              "mr",
+	Short:            "Add a Merge Request to Todo list",
+	Example:          `lab todo mr 1234                      #adds MR 1234 to user's Todo list`,
+	Long:             ``,
+	PersistentPreRun: LabPersistentPreRun,
+	Run: func(cmd *cobra.Command, args []string) {
+		rn, err := getRemoteName("")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		num, err := strconv.Atoi(args[0])
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		todoAddMergeRequest(rn, num)
+	},
+}
+
+func todoAddMergeRequest(remote string, mrNum int) {
+	todoID, err := lab.TodoMRCreate(remote, mrNum)
+	if err != nil {
+		if err.Error() == "EOF" {
+			log.Fatalf("Todo entry already exists for MR !%d", mrNum)
+		}
```

Comment:
Instead of checking for `EOF`, IMO it would be better to check for the `resp` return code in `lab.TodoMRCreate()` and return a valid error, similar to the approach we used when we created `ErrProjectNotFound` and others.

Reply:
Good point.  Fixed.